addons/mrp_bom_cost/models/product.py

```diff
@@ -11,11 +11,13 @@ class ProductTemplate(models.Model):
 
     def action_bom_cost(self):
         templates = self.filtered(lambda t: t.product_variant_count == 1 and t.bom_count > 0)
-        return templates.mapped('product_variant_id').action_bom_cost()
+        if templates.mapped('product_variant_id'):
```

Comment:
I think this is not necessary since `templates.mapped('product_variant_id').action_bom_cost()` returns nothing anyway.